php-checks/src/test/resources/checks/MultilineBlocksCurlyBracesCheck.php

```diff
@@ -2,56 +2,90 @@
 
 function nonCompliant() {
 
+  if ($condition) firstActionInBlock();
+
+  if ($condition) firstActionInBlock();  secondAction();  // Noncompliant
+  //              ^^^^^^^^^^^^^^^^^^^^^> ^^^^^^^^^^^^^^^
+
+  if ($condition);  secondAction();  // Noncompliant
```

Comment:
When everything is on one line, highlighting the previous one looks legit to me.
What will be highlighted if there is more than 2 statements on the same line? If they are all going to be highlighted, I would skip the first one again (the correct one).

Reply:
The message speaks about a group of n-statements where only the first will be executed conditionally. That's why I prefer highlighting the group. But to be more explicit, I'm going to add the secondary message "executed [conditionally | in a loop]" for the first statement, and "Not executed [conditionally | in a loop]" for the 3rd-to-last statement.

Reply:
Sounds like a good compromise to me.